recipes/orator/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "orator" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6fe7830c40f20e77929b80b741a3b9f2145634b5f411176ecad0e761fef26f55
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - orator = orator.commands.application:application.run
+
+requirements:
+  host:
+    - python =2.7|>=3.5,<3.7  # should be =2.7|>=3.5,<4 but backpack/pyaml clashes.
+    - pip
+    - poetry >=0.12a3
+  run:
+    - python =2.7|>=3.5,<3.7
+    - backpack >=0.1,<0.2
+    - blinker >=1.4,<2.0
+    - cleo >=0.6,<0.7
+    - inflection >=0.3,<0.4
+    - faker >=0.8,<0.9
+    - lazy-object-proxy >=1.2,<2.0
+    - pendulum >=1.4,<2.0
+    - pyaml >=16.12,<17.0
+    - pyyaml >=5.1,<6.0
+    - pygments >=2.2,<3.0
+    - simplejson >=3.10,<4.0
+    - six >=1.10,<2.0
+    - wrapt >=1.10,<2.0
+    - psycopg2 >=2.7,<3.0
+    - pymysql >=0.7,<0.8
+    - mysqlclient >=1.3,<2.0
+
+test:
+  imports:
+    - orator
+    - orator.commands
+    - orator.commands.migrations
+    - orator.commands.models
+    - orator.commands.seeds
+    - orator.connections
+    - orator.connectors
+    - orator.dbal
+    - orator.dbal.exceptions
+    - orator.dbal.platforms
+    - orator.dbal.platforms.keywords
+    - orator.dbal.types
+    - orator.events
+    - orator.exceptions
+    - orator.migrations
+    - orator.orm
+    - orator.orm.mixins
+    - orator.orm.relations
+    - orator.orm.scopes
+    - orator.pagination
+    - orator.query
+    - orator.query.grammars
+    - orator.query.processors
+    - orator.schema
+    - orator.schema.grammars
+    - orator.seeds
+    - orator.support
+    - orator.utils
+  commands:
+    - orator -h
+
+about:
+  home: https://orator-orm.com/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Looks like the license file is part of the PyPI tarball, `orator-0.9.9/LICENSE`, so please use `license_file: LICENSE` here and remove the additional LICENSE.txt file.

Reply:
You're right - that's been changed.